src/sage/categories/coxeter_groups.py

```diff
@@ -2368,19 +2383,19 @@ def bruhat_le(self, other):
             We now run consistency tests with permutations and
             :meth:`bruhat_lower_covers`::
 
-                sage: W = WeylGroup(["A",3])
-                sage: P4 = Permutations(4)
+                sage: W = WeylGroup(["A", 3])                                                                           # optional - sage.combinat sage.groups
+                sage: P4 = Permutations(4)                                                                              # optional - sage.combinat sage.groups
                 sage: def P4toW(w): return W.from_reduced_word(w.reduced_word())
-                sage: for u in P4:
+                sage: for u in P4:                                                                                      # optional - sage.combinat sage.groups
                 ....:     for v in P4:
                 ....:         assert u.bruhat_lequal(v) == P4toW(u).bruhat_le(P4toW(v))
 
-                sage: W = WeylGroup(["B",3])
-                sage: P = W.bruhat_poset() # This is built from bruhat_lower_covers
-                sage: Q = Poset((W, attrcall("bruhat_le")))                             # long time (10s)
-                sage: all( u.bruhat_le(v) == P.is_lequal(u,v) for u in W for v in W ) # long time  (7s)
+                sage: W = WeylGroup(["B", 3])                                                                           # optional - sage.combinat sage.groups
+                sage: P = W.bruhat_poset()  # This is built from bruhat_lower_covers                                    # optional - sage.combinat sage.groups sage.graphs
+                sage: Q = Poset((W, attrcall("bruhat_le")))                             # long time (10s)               # optional - sage.combinat sage.groups sage.graphs
+                sage: all(u.bruhat_le(v) == P.is_lequal(u,v) for u in W for v in W)     # long time  (7s)               # optional - sage.combinat sage.groups sage.graphs
```

Comment:
split so `#long time` fits in 80 columns, all the optional label can go to column 88.